docs/source/serving/high-availability.rst

```diff
@@ -0,0 +1,177 @@
+.. _skyserve-high-availability-controller:
+
+=================================
+High Availability SkyServe Controller
+=================================
+
+Overview
+--------
+By default, the SkyServe controller runs as a single instance (either a VM or a Kubernetes Pod). If this instance fails due to node issues, pod crashes, or other unexpected events, the service endpoint becomes unavailable until the controller is manually recovered or relaunched.
+
+To enhance resilience and ensure service continuity, SkyServe offers a High Availability (HA) mode for its controller. When enabled, the controller leverages Kubernetes capabilities to automatically recover from failures.
+
+Benefits of HA Controller:
```

Comment:
Use sentence case ("Benefits of HA controller", "How to enable HA mode", etc)
There should be a lint check for this - if you push a new commit hopefully the CI will trigger. Maybe it got confused because of the branch names.

Reply:
Good catch! Thanks for reminding.